client/src/Login/Login.js

```diff
@@ -44,12 +101,20 @@ export function Login() {
         {t('or')} <span className="uppercase font-bold">{t('login')}</span>
       </p>
 
-      {apiError && (
+      {apiError?.status && (
         <Alert
           className="mb-2"
-          message={apiError.message}
+          message={apiError?.message}
```

Comment:
If `apiError?.status` is truthy, `apiError` will have a `message` property. So, we don't need the `?`.